metricbeat/module/windows/perfmon/pdh_integration_windows_test.go

```diff
@@ -121,12 +125,41 @@ func TestNonExistingCounter(t *testing.T) {
 	}
 }
 
+func TestIgnoreNonExistentCounter(t *testing.T) {
+	config := Config{
+		CounterConfig:    make([]CounterConfig, 1),
+		IgnoreNECounters: true,
+	}
+	config.CounterConfig[0].InstanceLabel = "processor.name"
+	config.CounterConfig[0].MeasurementLabel = "processor.time.total.pct"
+	config.CounterConfig[0].Query = "\\Processor Information(_Total)\\not existing counter"
+	config.CounterConfig[0].Format = "float"
+	handle, err := NewPerfmonReader(config)
+
+	values, err := handle.Read()
+
+	time.Sleep(time.Millisecond * 1000)
```

Comment:
Any other way instead of using `Sleep` here? It makes the test suite much slower and has always potential to break.